src/app/app.component.html

```diff
@@ -71,39 +71,43 @@
               mat-button color="accent">#BlackLivesMatter</a>
           </span>
           <span class="d-none d-md-inline">
-            <button mat-button class="nav-button" routerLink="home" routerLinkActive="active">
+            <a mat-button class="nav-button" routerLink="home" routerLinkActive="active">
```

Comment:
I changed all of the `button` tags that were routerLinks to `a` tags, which gives you the URL preview in the bottom left corner of the browser.
![image](https://user-images.githubusercontent.com/9668239/84964477-36626a80-b0da-11ea-9703-72a807ceb28f.png)